gxa/src/main/java/uk/ac/ebi/atlas/ebeye/DifferentialExperimentContrastsTsvController.java

```diff
@@ -1,79 +0,0 @@
-package uk.ac.ebi.atlas.ebeye;
```

Comment:
We are still providing some sort of data for ebeye, aren't we? Is that coming straight from data prod?

Reply:
There are still two services we provide:
• Endpoint `/das/s4/features`
• The widget